lib/ansible/modules/database/postgresql/postgresql_pg_hba.py

```diff
@@ -191,7 +191,6 @@
 IPADDRESS_IMP_ERR = None
 try:
     import ipaddress
-    HAS_IPADDRESS = True
```

Comment:
I think you could actually get rid of `HAS_IPADDRESS` completely and just use `IPADDRESS_IMP_ERR`.
e.g 
```python
if IPADDRESS_IMP_ERR is not None:
    module.fail_json(msg=missing_required_lib('ipaddress'), exception=IPADDRESS_IMP_ERR)
```
rather than
```python
if not HAS_IPADDRESS:
    module.fail_json(msg=missing_required_lib('ipaddress'), exception=IPADDRESS_IMP_ERR)

Reply:
:+1: